techmatters/flex-plugins
# CHI-1106: Text transcript contact details

Primary reviewer: @GPaoloni 

## Description

* Adds a link to transcripts on contact details where available
* Add reservation to HRM contact data so transcripts / recordings can be retrieved

### Checklist
- N/A New tests added
- N/A Tested for call contacts

### Related Issues
CHI-1106

### Verification steps

* Create a webchat contact with some chat content
* View the contact
* Open the transcript and ensure it's contents are what you expect
